pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1239,6 +1240,32 @@ func TestSchedulerCache_UpdateSnapshot(t *testing.T) {
 		podsWithAffinity = append(podsWithAffinity, pod)
 	}
 
+	// Add a few of pods with PVC, starting on the third node.
+	podsWithPVC := []*v1.Pod{}
```

Comment:
It's more conventional:

```suggestion
	var podsWithPVC []*v1.Pod
```

Reply:
Sure, and fixed in all the other places in this file.